internal/rbd/controllerserver.go

```diff
@@ -881,9 +881,14 @@ func (cs *ControllerServer) checkErrAndUndoReserve(
 	}
 
 	if errors.Is(err, ErrImageNotFound) {
-		err = rbdVol.ensureImageCleanup(ctx)
-		if err != nil {
-			return nil, status.Error(codes.Internal, err.Error())
+		// if we get here, we need to connect the rbdVol so an ioctx can be created
+		notFoundErr := rbdVol.Connect(cr)
```

Comment:
i looked at the code i see we are doing the Connect operation in the caller? can you please check if the connection is required here (let me know if am missing anything)?

Reply:
indeed, the `.Connect()` should not be needed